src/main/java/pages/AllRatingsPage.java

```diff
@@ -8,19 +8,16 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
 import java.time.Duration;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public class AllRatingsPage extends BasePage {
-
-    public AllRatingsPage(WebDriver driver) {
-        super(driver);
-    }
-
+public class AllRatingsPage {
+    WebDriver driver;
```

Comment:
Where is the BasePage? Why it is deleted?
I've told you to inherit all Page Objects from BasePage.